### DISCUSSION TITLE
Contribution: Icons

### DISCUSSION THREAD

**Discussion by A4D4R** [Zodsmar/SeedSearcherStandaloneTool#41] [Show and tell]
While I was making a package for [AUR](https://aur.archlinux.org/), I found out that SASSA doesn't have any icon (aside from the  one with blue background, white text Sassa .ico file). So, here is my take on the icons.

Since it was using Amidst, I decided to tweak the amidst icon, adding seed on top (The mossy stone bricks texture is straight from Minecraft asset file, seed texture is SVG on top so its partially scalable).

Feedback is welcome as to improve on this or you can provide me with any idea you have, I'll try to create it.

License: Use as you like. The background mossy stone brick is from Minecraft so I cant just slap any license on top of it.

Files:
- .png - 16x16 32x32 48x48 64x64 128x128 256x256
- ~.ico - 256x256~ _(Looks like I cannot upload .ico on github but you can just use online coverter to change 256x .png to .ico format)_

![sassa-16x16](https://user-images.githubusercontent.com/29174026/85420392-dc0e5180-b592-11ea-80d6-34184ce48523.png)
![sassa-32x32](https://user-images.githubusercontent.com/29174026/85420402-df094200-b592-11ea-8ed0-ca9e0758030c.png)
![sassa-48x48](https://user-images.githubusercontent.com/29174026/85420404-dfa1d880-b592-11ea-914d-b778a8edc5d7.png)
![sassa-64x64](https://user-images.githubusercontent.com/29174026/85420407-e03a6f00-b592-11ea-891b-a7e38981607c.png)
![sassa-128x128](https://user-images.githubusercontent.com/29174026/85420409-e0d30580-b592-11ea-8faf-466432ed2cfe.png)
![sassa-256x256](https://user-images.githubusercontent.com/29174026/85420412-e16b9c00-b592-11ea-8a9b-05d0fcc9cbf1.png)

**Comment by Zodsmar** [comment]
Thanks! I like it a lot. It's definitely something useful that the tool needed. The next version will have nether and end searching adding. So I'm thinking do you wanna leave the top half mossy brick and the two bottom sections left and right change those corners to maybe a nether block of some kind and an end block of some kind to show that it searchers all dimensions?

**Comment by A4D4R** [comment]
Let me try that out, will Post 256x here, if you like it then will resize it to other sizes. 

Also instead of using Minecraft textures, I'll try to make my own (haven't done that in a while) so you can slap some free license compatible with GPL.

**Comment by A4D4R** [comment]
Tried making custom graphics instead of using minecraft assets, but didn't work too well. Maybe some other people doing textures can help. Anyway, here is the Overworld, nether and end "Seed".

![Seed Search Standalone](https://user-images.githubusercontent.com/29174026/85446636-687d3c00-b5b4-11ea-9f6a-59fafed8393c.png)

Edit: on second thought, i think i should lower the grass on top of netherrack and endstone a bit, current one looks a bit weird at edges of netherrack and endstone. Will be much easier if we got a discord server or something for seeds.

**Comment by Zodsmar** [comment]
There is a discord server for the r/minecraftseeds already but if people want a seed searcher discord I don't mind making one. Also, I do like the texture. If you wanna make those changes you mentioned about lowering the grass go for it!

**Comment by Zodsmar** [comment]
Updated the icon. 575b324
